RELEASE_NOTES.md

```diff
@@ -1,78 +1,45 @@
-# Cosmos SDK v0.40.0 "Stargate" Release Notes
+# Cosmos SDK v0.40.1 "Stargate" Release Notes
 
-This release introduces several new important updates to the Cosmos SDK. The release notes below provide an overview of
-the larger high-level changes introduced in the v0.40 (aka Stargate) release series.
+This is a bugfix-only release to the *Cosmos SDK 0.40* "Stargate" release series. No breaking changes are introduced, thus no migration should be needed.
```

Comment:
Seems a bit odd to state it's a bugfix-only release, if we add an improvement in the changelog.